yarn.config.cjs

```diff
@@ -835,3 +837,60 @@ async function expectReadme(workspace, workspaceBasename) {
     );
   }
 }
+
+// A promise resolving to the codeowners file contents
+let cachedCodeownersFile;
+
+/**
+ * Expect that the workspace has a codeowner set, and that the CHANGELOG.md and
+ * package.json files are co-owned with the wallet framework team.
+ *
+ * @param {Workspace} workspace - The workspace to check.
+ * @param {string} workspaceBasename - The name of the workspace.
+ * @returns {Promise<void>}
+ */
+async function expectCodeowner(workspace, workspaceBasename) {
+  if (!cachedCodeownersFile) {
+    cachedCodeownersFile = readFile(
+      resolve(__dirname, '.github', 'CODEOWNERS'),
+      'utf8',
+    );
+  }
+  const codeownersFile = await cachedCodeownersFile;
+  const codeownerRules = codeownersFile.split('\n');
+
+  const packageCodeownerRule = codeownerRules.find((rule) =>
+    rule.startsWith(`/packages/${workspaceBasename}`),
```

Comment:
Perhaps there should be a space at the end of the path, otherwise we won't catch the case where there is a rule for the CHANGELOG but not one for the rest of the package files 
``` suggestion
    rule.startsWith(`/packages/${workspaceBasename} `),
```

Reply:
Good point. Yes that sounds like a good solution.